charts/keda/values.yaml

```diff
@@ -0,0 +1,7 @@
+enabled: true
+
+keda:
+  image:
+    # Enable KEDA POC in cloud
+    keda: "dimberman/keda:master"
+    metricsAdapter: "dimberman/keda-metrics-adapter:master"
```

Comment:
I'm still working on this, I will get it vendored before we merge

Reply:
I can't find what you built your image from... can you PR to ap-vendor please @dimberman

Reply:
@dimberman @sjmiller609 these are being published to dockerhub now.

Reply:
@schnie I've changed it to point to our docker image